internal/service/resourcepolicy/resource_schema.go

```diff
@@ -0,0 +1,121 @@
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func ResourceSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"created_by_user": schema.SingleNestedAttribute{
+				Description: "The user that last updated the atlas resource policy.",
+				Computed:    true,
+				Attributes: map[string]schema.Attribute{
+					"id": schema.StringAttribute{
+						Description: "Unique 24-hexadecimal character string that identifies a user.",
+						Computed:    true,
+					},
+					"name": schema.StringAttribute{
+						Description: "Human-readable label that describes a user.",
+						Computed:    true,
+					},
+				},
+			},
+			"created_date": schema.StringAttribute{
+				Description: "Date and time in UTC when the atlas resource policy was created.",
+				Computed:    true,
+			},
+			"id": schema.StringAttribute{
+				Description: "Unique 24-hexadecimal digit string that identifies an atlas resource policy.",
+				Computed:    true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"last_updated_by_user": schema.SingleNestedAttribute{
+				Description: "The user that last updated the atlas resource policy.",
+				Computed:    true,
+				Attributes: map[string]schema.Attribute{
+					"id": schema.StringAttribute{
+						Description: "Unique 24-hexadecimal character string that identifies a user.",
+						Computed:    true,
+					},
+					"name": schema.StringAttribute{
+						Description: "Human-readable label that describes a user.",
+						Computed:    true,
+					},
+				},
+			},
+			"last_updated_date": schema.StringAttribute{
+				Description: "Date and time in UTC when the atlas resource policy was last updated.",
+				Computed:    true,
+			},
+			"name": schema.StringAttribute{
+				Description: "Human-readable label that describes the atlas resource policy.",
+				Optional:    true,
+			},
+			"org_id": schema.StringAttribute{
+				Description: "Unique 24-hexadecimal digit string that identifies the organization that contains your projects. Use the [/orgs](#tag/Organizations/operation/listOrganizations) endpoint to retrieve all organizations to which the authenticated user has access.",
+				Required:    true,
+			},
+			"policies": schema.ListNestedAttribute{
+				Description: "List of policies that make up the atlas resource policy.",
+				Required:    true,
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"body": schema.StringAttribute{
+							Description: "A string that defines the permissions for the policy. The syntax used is the Cedar Policy language.",
+							Required:    true,
+						},
+						"id": schema.StringAttribute{
+							Description: "Unique 24-hexadecimal character string that identifies the policy.",
+							Computed:    true,
+						},
+					},
+				},
+			},
+			"version": schema.StringAttribute{
```

Comment:
should we use this plan modifier for this computed attributes as well? or can this possibly change?

Reply:
Good idea. Added ✅